### ISSUE TITLE
[Feature Request] Ability to upload a pool of officials

### ISSUE THREAD

**Issue by bullseye555** [rollerderby/scoreboard#517]
Discussing with other officials here in Australia - the idea was floated that a League will have a regular pool of officials and it would be useful to be able to upload (and/or input) a 'team' or pool of Officials that will be used for a Home Team season. This would allow the officials to be loaded at the start of the season, and a game started with the Home Team's, and the officials populated from drop-downs in the _StatsBook_ tab

@JeneralPain - Can you please add labels etc?

**Comment by frank-weinberg** [comment]
The main feature I'm envisioning for 5.1 is management of tournaments/series of games. I'll take this request into account when designing that.

(But I don't have an estimate yet on when I will get to work on that.)

**Comment by bullseye555** [comment]
Thanks for the update @frank-weinberg

**Comment by phillipstrong9** [comment]
Can we bring this back to the top of the requests?

**Comment by frank-weinberg** [comment]
I would love to have this feature myself. But unfortunately I don't see any way to get it implemented in time for v2023, even as a standalone feature without the other tournament features I'm envisioning.

**Comment by spikenheimer** [comment]
fwiw someone on FB pointed out that copy paste from columnar data ala a spreadsheet magics this already in the data entry (tested on v2023.0) in a similar fashion to team skater entry.  using Role / name / league as the columns works pretty well. only minor quibble is the camel casing for Role must match the pull downs or role will be listed as Other with the role data listed as pasted. 

tldr: copy/paste workaround might solve main need vs managing a internal catalog of officials
